src/EventEdition/InfoPanel.vala

```diff
@@ -264,6 +285,31 @@ public class Maya.View.EventEdition.InfoPanel : Gtk.Grid {
                 Gtk.TextBuffer buffer = new Gtk.TextBuffer (null);
                 buffer.text = property.get_comment ();
                 comment_textview.set_buffer (buffer);
+
+                // Identify links in the description and apply a tag to style them
+                var link_regex = new GLib.Regex ("https?://[\\w.-]*(\\/[\\w-\\d]*)*");
```

Comment:
The compiler also gives a warning of an unhandled `GLib.RegexError` on this line.